test/integration/updating_permissions_for_apps_with_many_permissions_test.rb

```diff
@@ -162,14 +152,13 @@ def assert_permissions_unchanged
       should "reset the value of the select element when it no longer matches what's shown in the autocomplete input" do
         assert_select_permission_to_grant_with_javascript
 
-        @autocomplete_input.fill_in with: "addin"
-        autocomplete_option = find(".autocomplete__option")
-
-        assert_equal "addin", @autocomplete_input.value
-        assert_equal @new_permission_to_grant.name, autocomplete_option.text
-        assert_equal "", @select_element.value
+        assert_resets_select_when_desynced_with_autocomplete(
```

Comment:
If we are going to go down a more abstracted approach with our tests, could we make method names for our helpers even more human readable? I know we're following the lead taken by Minitest here, but I'm not the biggest fan of the brevity of Minitest's DSL.

Also, forgive my ignorance but I'm not sure what desynced means!